eapp/public/js/ndarImport.js

```diff
@@ -15,96 +15,112 @@ $.fn.select2.defaults.set( "theme", "bootstrap" );
-    for (let i=0;i<nsources.length;i++){
-        $("#ndar-source").append('<option value="'+ nsources[i]+'">'+ nsources[i] +'</option>')
-        //count++
-    }
-  }
-})
-
-/*
-* NDAR-CATEGORIES
-*/
-$.ajax({
+//$.ajax({
+//  type: "GET",
+//  url: "http://localhost:3000/ndar-types",
+//  accept: "application/json",
+//  success: function(data){
+//    console.log('ndar-types:success')
+//    let dE = JSON.parse(data)
+//    ntypes = dE.list
+//    $("#ndar-type").select2()
+//    for (let i=0;i<ntypes.length;i++){
+//        console.log(ntypes[i])
+//        $("#ndar-type").append('<option value="'+ ntypes[i]+'">'+ ntypes[i] +'</option>')
+//    }
+//  }
+//})
+//
+///*
+//* NDAR-SOURCES
+//*/
+//$.ajax({
+//  type: "GET",
+//  url: "http://localhost:3000/ndar-sources",
+//  accept: "application/json",
+//  success: function(data){
+//    console.log('ndar-sources:success')
+//    let dE = JSON.parse(data)
+//    nsources = dE.list
+//    $("#ndar-source").select2()
+//    for (let i=0;i<nsources.length;i++){
+//        $("#ndar-source").append('<option value="'+ nsources[i]+'">'+ nsources[i] +'</option>')
+//        //count++
+//    }
+//  }
+//})
+//
+///*
+//* NDAR-CATEGORIES
+//*/
+//$.ajax({
+//  type: "GET",
+//  url: "http://localhost:3000/ndar-categories",
+//  accept: "application/json",
+//  success: function(data){
+//    console.log('ndar-cat:success')
+//    let dE = JSON.parse(data)
+//    categories = dE.list
+//    $("#ndar-cat").select2()
+//    for (let i=0;i<categories.length;i++){
+//        $("#ndar-cat").append('<option value="'+ categories[i]+'">'+ categories[i] +'</option>')
+//    }
+//  }
+//})
+//
+///*
+//* Data Dictionaries
+//*/
+//function getDataForms(e1){
+//  e1.preventDefault()
+//  $("#btn-dataForms").remove()
+//  nparamObj['type'] = $("#ndar-type").val()
+//  nparamObj['source'] = $("#ndar-source").val()
+//  nparamObj['category'] = $("#ndar-cat").val()
+//  console.log(nparamObj)
+//
+//  $.ajax({
+//    type: "POST",
+//    url: "http://localhost:3000/ndar-terms/forms",
+//    contentType: "application/json",
+//    data: JSON.stringify(nparamObj),
+//    success: function(data){
+//      console.log('get forms: success')
+//      let dE = JSON.parse(data)
+//      console.log(dE)
+//      /*$( "#ndar-forms" ).select2({
+//          theme: "bootstrap"
+//      });*/
+//
+//      $("#ndar-dd").append('<select class="form-control" id="ndar-forms">\
+//          <option value="ddform">Select a form</option>\
+//          </select>')
+//
+//      $("#ndar-forms").select2()
+//      for (let i=0;i<dE.length;i++){
+//          $("#ndar-forms").append('<option value="'+ dE[i].shortName+'">'+ dE[i].title +'</option>')
+//      }
+//    }
+//  })
+//
+//}
+
+  /*New consolidated data dicitonary get request*/
+  $.ajax({
   type: "GET",
-  url: "http://localhost:3000/ndar-categories",
+  url: "http://localhost:3000/ndar-terms/forms",
   accept: "application/json",
   success: function(data){
```

Comment:
indentation should be 2 spaces.  Use this style guide (recommended by Electron) as reference: https://www.npmjs.com/package/standard